context-monitoring/src/main/java/de/atb/context/services/wrapper/AmIMonitoringDataRepositoryServiceWrapper.java

```diff
@@ -163,7 +164,7 @@ public final Type getMonitoringData(final BusinessCase businessCase, final Class
     @Override
     public final synchronized ResultSet executeSparqlSelectQuery(final BusinessCase businessCase, final String query) throws ContextFault {
         String resultSetXmlString = this.service.executeSparqlSelectQuery(businessCase, query);
-        return ResultSetFactory.fromXML(resultSetXmlString);
+        return ResultSetFactory.fromXML(IOUtils.toInputStream(resultSetXmlString));
```

Comment:
`CharSet` missing here?

Reply:
added UTF-8 charset

Reply:
It's not in the current version: https://github.com/eclipse-researchlabs/smartclide-context/blob/replcaeOldJenaLibs/context-monitoring/src/main/java/de/atb/context/services/wrapper/AmIMonitoringDataRepositoryServiceWrapper.java

maybe missed a commit?